### PR TITLE
fix display of code blocks in index view

### PR DESCRIPTION
**Pull request by royneary**
If readme files contain markup, they weren't displayed correctly in the
index view (no padding, no grey box).

The fix contains two changes:
- Use `force_unicode` on the readme's filename
- use the CSS properties for both blobview and non-blobview

### CODE REVIEW

**Comment by royneary** [comment]
- On the left: The klaus.git index page without the fix (code boxes don't have padding, borders and background):
- On the right: The same page with the fix

klaus

File: klaus/views.py [hunk 1]
```diff
@@ -274,16 +274,15 @@ def make_template_context(self, *args):
             (readme_filename, readme_data) = self._get_readme()
         except KeyError:
             self.context.update({
-                'is_markup': None,
+                'is_markup': False,
```
Comment (jonashaag) [inline klaus/views.py]: Why this change?

Comment (royneary) [reply klaus/views.py]: It was an experiment before I found the fix. So the change is not necessary but I left it there because `is_markup` sounds like a boolean (what would it mean if it was None?). But AFAIK None and False are equivalient in boolean expressions. You decide.

**Comment by jonashaag** [comment]
Thanks!

**Comment by jonashaag** [comment]
IIRC the None value there actually had some semantic (at least readability) meaning, so I've reverted the change.
